.hadolint.yaml

```diff
@@ -8,6 +8,5 @@ ignored:
 # in most cases, packages are installed via scripts and those don't get checked
 # (albeit, ubuntu repos tend to keep old versions around)
 - DL3008
-# this one just needs to be set, but since `FROM ${BASE_IMAGE}` cannot be parse,
-# there isn't a way to leverage this feature anyway
-allowedRegistries: []
```

Comment:
Note: removing `allowedRegistries` as it does not appear to be necessary anymore (and that parameter got renamed to `trustedRegistries` a couple years ago anyway, see https://github.com/hadolint/hadolint/pull/600)